libbeat/docs/dashboards.asciidoc

```diff
@@ -6,7 +6,14 @@ the perfect dashboards for your needs. But this flexibility can be a bit
 overwhelming at the beginning, so we have created some sample dashboards to get you
 started and to demonstrate what is possible based on the Beats data.
 
-NOTE: The Beats dashboards require Kibana 4 or later.
+NOTE: The Beats dashboards require Kibana 5 or later.
```

Comment:
@monicasarbu OK, so it sounds like you want me to:
- revert the original statement on line 9 (to say "The Beats dashboards require Kibana 4 or later" )
- remove the note under "Known Issue in 5.0.0-alpha5". I thought there was an issue specific to our dashboards (and that users who have an earlier version of the ES alpha installed would need to move to alpha5 to get our dashboards to work). But it sounds like this is a Kibana + ES issue and should probably be documented in the release notes for those products. 

Is that right?